spark/src/main/scala/geotrellis/spark/tiling/MapKeyTransform.scala

```diff
@@ -95,12 +103,16 @@ class MapKeyTransform(val extent: Extent, val layoutCols: Int, val layoutRows: I
     (tcol.floor.toInt, trow.floor.toInt)
   }
 
-  def apply[K: SpatialComponent](key: K): Extent = {
-    apply(key.getComponent[SpatialKey])
-  }
+  def keyLikeToExtent[K: SpatialComponent](key: K): Extent = apply(key)
```

Comment:
Hmm, this could actually take care of the SpatialKey case just fine. Do you think its more clear to have an explicit function not based on `SpatialComponent`?

Reply:
I think a buy that, what are some of the impacted lines in our codebase for reference ?